lyluatex.sty

```diff
@@ -60,6 +61,12 @@
 }
 \catcode`-=12
 
+\directlua{
+    if ly.get_option('verbatim-highlight') then
+        tex.sprint([[\detokenize{\RequirePackage{minted}}]])
+    end
+}
+
 \newcommand{\ly@setunits}{%
```

Comment:
I don't think this is the correct approach because it will fail when the user applies `verbatim-highlight` as a local option only. 

I suggest moving that test to the function where verbatim is actually *used*.
It could be discussed whether it is worse to repeatedly write `\RequirePackage` into the TeX code or testing each time if `minted` is already loaded.

Reply:
OK, I'll look at this too.